Makefile

```diff
@@ -97,16 +97,23 @@ azurefile-windows:
 	CGO_ENABLED=0 GOOS=windows go build -a -ldflags ${LDFLAGS} -o _output/azurefileplugin.exe ./pkg/azurefileplugin
 
 .PHONY: container	
-container: azurefile	
-	docker build --no-cache -t $(IMAGE_TAG) -f ./pkg/azurefileplugin/Dockerfile .
+container:	
+	docker build --no-cache --build-arg LDFLAGS=$(LDFLAGS) -t $(IMAGE_TAG) -f ./pkg/azurefileplugin/Dockerfile .
 
 .PHONY: azurefile-container
 azurefile-container:
 ifdef CI
 	az acr login --name $(REGISTRY_NAME)
-	make azurefile azurefile-windows
-	az acr build --registry $(REGISTRY_NAME) -t $(IMAGE_TAG)-linux-amd64 -f ./pkg/azurefileplugin/Dockerfile --platform linux .
-	az acr build --registry $(REGISTRY_NAME) -t $(IMAGE_TAG)-windows-1809-amd64 -f ./pkg/azurefileplugin/Windows.Dockerfile --platform windows .
+	# make azurefile azurefile-windows
+	# az acr build --registry $(REGISTRY_NAME) -t $(IMAGE_TAG)-linux-amd64 -f ./pkg/azurefileplugin/Dockerfile --platform linux .

```

Comment:
remove the comments here?

Reply:
i will i was just trying to get the ci to pass first and then ill clean up the pr commits